guides/source/action_cable_overview.md

```diff
@@ -542,7 +542,7 @@ test:
 
 production:
   adapter: redis
-  url: redis://10.10.3.153:6381
+  url: redis://user:secret@localhost:6381/12?foo=bar&qux=baz
```

Comment:
Redis isn't supposed to receive a username at the moment. AUTH only accepts a password.